internal/fileutils/fileutils.go

```diff
@@ -442,6 +442,21 @@ func IsEmptyDir(path string) (bool, error) {
 	return (len(files) == 0), nil
 }
 
+// IsEmptyDirs returns true if the directory at the provide path has no files
+// within it or any of its sub directories
+func IsEmptyDirs(path string) bool {
+	empty := true
+	filepath.Walk(path, func(p string, _ os.FileInfo, _ error) error {
+		if !IsDir(p) {
```

Comment:
I think, even if `p` is a directory, we should set `empty = false`.  But also: the callback can just always *return* `true`.  Once it is called once for a file in that directory, we know that it is not empty.

Reply:
This function is no longer needed due to your suggestion below 👍